lib/sidekiq/statistic/web_extension.rb

```diff
@@ -28,7 +28,7 @@ def self.registered(app)
 
         app.get '/statistic' do
           statistic = Sidekiq::Statistic::Workers.new(*calculate_date_range(params))
-          @workers = statistic.display
+          @Allworkers = statistic.display
```

Comment:
Hey @GQuirino, I was seeing the changed files, and I think the array variable name is not in the Ruby convention, I think @all_workes could be better

Reply:
You're right, sorry man.